tools/build-manylinux.sh

```diff
@@ -44,7 +44,7 @@ fi
 #
 
 echo "# Installing basic system dependencies"
-yum install -y zlib-devel gcc-c++
+yum install -y zlib-devel gcc-c++ƒ
```

Comment:
this looks weird

Reply:
Interesting I thought I pushed the removal of this prior to opening a pr

Reply:
Good thing there is a web page showing what has actually been pushed.
This one
;)